### ISSUE TITLE
[Question]  about 1.6 training

### ISSUE THREAD

**Issue by nixsui** [haotian-liu/LLaVA#1331]
First of all, thank you for sharing such excellent work and code. I have some questions about the training of 1.6. I tried to modify the code to train version 1.6 myself. I have some doubts about some details. 
1. Regarding the issue of resolution, I saw in the report that it has reached 672, is this dynamic high resolution? 
<img width="929" alt="image" src="https://github.com/haotian-liu/LLaVA/assets/42858061/fe782376-2042-4dbc-94fc-a7cd0c25d95d">

2. I noticed in the report that the full model was fine-tuned in the second stage. Does this include the image encoder?
<img width="671" alt="image" src="https://github.com/haotian-liu/LLaVA/assets/42858061/b0df2588-368d-45e2-a8dd-cc1eded8bcd8">

**Comment by nixsui** [comment]
3. Was dynamic high resolution also used in the pretraining stage?
![image](https://github.com/haotian-liu/LLaVA/assets/42858061/0c22c068-12b6-4974-9b71-f0cc4fd88832)

**Comment by nixsui** [comment]
1、During my debugging, I found that the token length of dynamic high resolution has reached 2000+, so I believe the authors did not use a larger resolution.
2、I looked at the weights of LLaVA 1.6 and saw that the config includes the learning rate of the vision encoder, so I believe that it includes the vision encoder here.
3、I will conduct the experiment for verification and synchronize the results afterward.

**Comment by BlueBlueFF** [comment]
> 1、During my debugging, I found that the token length of dynamic high resolution has reached 2000+, so I believe the authors did not use a larger resolution. 2、I looked at the weights of LLaVA 1.6 and saw that the config includes the learning rate of the vision encoder, so I believe that it includes the vision encoder here. 3、I will conduct the experiment for verification and synchronize the results afterward.

Any update？
